code/game/objects/items/weaponry.dm

```diff
@@ -974,13 +974,23 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 /obj/item/club/pre_attack(atom/A, mob/living/user, params)
 	force = initial(force)
 	armour_penetration = initial(armour_penetration)
-	if(isstructure(A) || ismachinery(A) || isturf(A))
-		force *= 4
+	if(isstructure(A))
+		force *= 2
 		armour_penetration += 50
 		// To prevent unnecessary force string calculation (we want this to be treated
 		// as if it wasn't changed)
 		last_force_string_check = force
-	return ..()
+		return ..()
+	else if(ismachinery(A))
+		force *= 2.5
+		armour_penetration += 50
+		last_force_string_check = force
+		return ..()
```

Comment:
Why is machinery different to structure?

Reply:
Because machinery usually has a bit more health. This felt right in testing!